Libraries/LibWeb/SVG/SVGElement.cpp

```diff
@@ -32,75 +33,92 @@ void SVGElement::initialize(JS::Realm& realm)
 }
 
 struct NamedPropertyID {
-    NamedPropertyID(CSS::PropertyID property_id)
+    NamedPropertyID(CSS::PropertyID property_id, Vector<FlyString> supported_elements = {})
         : id(property_id)
         , name(CSS::string_from_property_id(property_id))
+        , supported_elements(move(supported_elements))
     {
     }
 
     CSS::PropertyID id;
     StringView name;
+    Vector<FlyString> supported_elements;
 };
 
-static Array const attribute_style_properties {
```

Comment:
I had to move this list into a method to be able to use `SVG::TagNames::*`. Leaving the list here caused the FlyStrings to appear to be uninitialized.

I'm assuming this is because the static variable is initialized before the FlyString itself, but I'm not sure.